docs/ECLStandardLibraryReference/SLR-Mods/SprayVariable.xml

```diff
@@ -29,16 +31,19 @@
   <emphasis>allowoverwrite</emphasis> <emphasis role="bold">] , [</emphasis>
   <emphasis>replicate</emphasis> <emphasis role="bold">] , [</emphasis>
   <emphasis> compress </emphasis> <emphasis role="bold">] , [</emphasis>
-  <emphasis> sourceCsvEscape </emphasis> <emphasis role="bold">])</emphasis>
-  <emphasis></emphasis></para>
+  <emphasis> sourceCsvEscape </emphasis> <emphasis role="bold">]</emphasis>,
```

Comment:
It seems something wrong. My 'HPCC-16630 Add expiry parameter to spray commands' merged into 7.0.0 therefore the expireDays doesn't available in 6.4.0. 
@richardkchapman @ghalliday I can see two solutions:
1. Merge https://github.com/hpcc-systems/HPCC-Platform/pull/9470 down to 6.4.0
2. Retarget this PR to 7.0.0